source/features/comment-shortcuts.tsx

```diff
@@ -0,0 +1,92 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+const shortcutKeys = new Set(['e', 'd', 'h', 'j', 'k', '+', '-', 'n']);
+const shortcutClass: Record<string, string> = {
+	e: '.rgh-edit-comment, [aria-label="Edit comment"]',
+	d: '[aria-label="Delete comment"]',
+	h: '[aria-label="Hide comment"]',
+	'+': '[data-reaction-label="+1"]',
+	'-': '[data-reaction-label="-1"]'
+};
+
+function triggerShortcut(shortcut: string) {
+	const selectedComment = select(location.hash)!;
+	select<HTMLButtonElement>(shortcutClass[shortcut], selectedComment)?.click();
+}
+
+function focusComment({delegateTarget: comment}: delegate.Event<MouseEvent, HTMLButtonElement>): void {
+	if (select.exists(':target')) {
+		console.log('I dont work correctly');
+	}
+
+	history.replaceState({}, document.title, select<HTMLAnchorElement>('a.js-timestamp', comment)!.hash);
+}
+
+function init(): void {
+	delegate(document, '.timeline-comment.unminimized-comment, .review-comment', 'click', focusComment);
+	document.addEventListener('keypress', event => {
+		if (!shortcutKeys.has(event.key) || event.target instanceof HTMLTextAreaElement || event.target instanceof HTMLInputElement) {
+			return;
+		}
+
+		event.preventDefault();
+
+		if (['j', 'k'].includes(event.key)) {
+			const items = select.all<HTMLAnchorElement>('.timeline-comment-group[id^="issue"]:not([href])');
```

Comment:
This could be changed to `.js-targetable-element`, that way, you can target review comments too.

PS: You may have to filter out some items that you do not want to target.

/cc @fregante

Reply:
My issue was that how do I stay in the same review? (I would not want it to go to the next review as that could be totally unrelated)

Reply:
We do get all comments in order when querying DOM, we can treat everything as a "comment".

How exactly is this an issue, can you please explain any unwanted behavior you are seeing here.

Reply:
> My issue was that how do I stay in the same review?

The answer is that you don't. j/k only mean previous/next on the current page. We're not going to implement boundaries.

If it's a comment, we should focus it.

Reply:
But skip I wil skip the minimized comments/reviews